x-pack/plugins/file_data_visualizer/public/application/components/stats_table/components/field_data_row/boolean_content_preview.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useMemo } from 'react';
+// import { EuiDataGridColumn } from '@elastic/eui';
+import { OrdinalChartData } from './field_histograms';
+// import { ColumnChart } from '../../../../../../components/data_grid/column_chart'; // TODO copy component
+import { FieldDataRowProps } from '../../types';
+import { getTFPercentage } from '../../utils';
+
+export const BooleanContentPreview: FC<FieldDataRowProps> = ({ config }) => {
+  const chartData = useMemo(() => {
+    const results = getTFPercentage(config);
+    if (results) {
+      const data = [
+        { key: 'true', key_as_string: 'true', doc_count: results.trueCount },
+        { key: 'false', key_as_string: 'false', doc_count: results.falseCount },
+      ];
+      return { id: config.fieldName, cardinality: 2, data, type: 'boolean' } as OrdinalChartData;
+    }
+  }, [config]);
+  if (!chartData || config.fieldName === undefined) return null;
+
+  // const columnType: EuiDataGridColumn = {
+  //   id: config.fieldName,
+  //   schema: undefined,
+  // };
+  // const dataTestSubj = `mlDataGridChart-${config.fieldName}`;
+
+  return (
+    <></>
```

Comment:
Should we put this ColumnChart back?

Reply:
As mentioned in the PR description, enabling this chart involves copying over a lot of ML code, so has been disabled for the time being.
I'm not too sure if this chart was working correctly in the first place. With this code commented out boolean fields work correctly.

![image](https://user-images.githubusercontent.com/22172091/114888239-f30f8000-9e00-11eb-9123-3d1b8b8c7616.png)

Reply:
I think that `boolean_content_preview` here is for the preview/collapsed row. But I understand it can be disabled for now.

<img width="1782" alt="Screen Shot 2021-04-15 at 09 51 08" src="https://user-images.githubusercontent.com/43350163/114889884-238af600-9dd0-11eb-9dbd-725fdd853c8f.png">

Reply:
ah ok, yes it can be re-enabled with a bit of work later on